gravitee-apim-portal-webui-next/src/components/banner/banner.component.scss

```diff
@@ -18,6 +18,7 @@
 
 .banner {
   --mdc-elevated-card-container-color: #{$banner-background-color};
+  --mdc-elevated-card-container-shape: 8px;
```

Comment:
I was thinking of reusing this banner as well for the Api details view. (It has the Api description in it with a lightbulb icon.) Did you also take that into account with this line?

Reply:
Hm, I see there is 12px, I'm deleting it for now